### PR TITLE
Update alexa-remote.js (acceptLanguage)

### PR DESCRIPTION
**Pull request by jeremyarnoux**
problem with languages:
replacing de-DE with this._options.acceptLanguage
and initialization of acceptLanguage
thanks

### CODE REVIEW

File: alexa-remote.js [hunk 1]
```diff
@@ -2910,7 +2910,7 @@ class AlexaRemote extends EventEmitter {
         const operationPayload = {
             'deviceType': dev.deviceType,
             'deviceSerialNumber': dev.serialNumber,
-            'locale': 'de', // TODO!!
+            'locale': this._options.acceptLanguage, // TODO!!
```
Comment (Apollon77) [inline alexa-remote.js:2913]: Did you tested that this works becazsuse here no locale is used but "just" a language code ... ?

**Review by Apollon77 — COMMENTED** [review]
see comment

File: alexa-remote.js (hunk 1 above)
Comment (Apollon77) [reply alexa-remote.js:2913]: Ok I wait till you have tested otr I found time

Comment (jeremyarnoux) [reply alexa-remote.js:2913]: I tested with fr and de, no change... Alexa speaks in French (because configuration in fr for my tests)

**Pull request closed without merging**